app/javascript/stylesheets/application.scss

```diff
@@ -6,6 +6,12 @@
 
 @import "@bigbinary/neetoui";
 
+// variables
```

Comment:
No need for this comment. The file names(or namespaces) are pretty self explanatory at the moment.